internal/cmd/organization/command_describe.go

```diff
@@ -0,0 +1,42 @@
+package organization
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+)
+
+type out struct {
+	Id        string `human:"ID" serialized:"id"`
+	Name      string `human:"Name" serialized:"name"`
+}
+
+func (c *command) newDescribeCommand() *cobra.Command {
```

Comment:
I think we should consider calling the list endpoint here so we can allow the user to pass in the `<id>`... any reason not to?

Reply:
CDMUM recommended that we not do this without a compelling reason. The list endpoint is the only exception to the general rule of not allowing access to other orgs.

Reply:
Maybe the idea is that in the future, the describe endpoint will return sensitive data that's not returned by the list endpoint?

If that's the case, the describe endpoint could just omit those sensitive fields if it's not the current org...

This seems really inconsistent & bad UX!

Reply:
Hmm, checking the thread again this discussion was actually about the behavior of the describe endpoint; and how we did not want to design it to allow access for different orgs. As far as the CLI is concerned I think that might have been discussed in a meeting instead.

Well, I can ask them if they're ok with the CLI working around it via the list endpoint.